src/persist-client/src/cfg.rs

```diff
@@ -176,7 +177,6 @@ impl PersistConfig {
     }
 
     /// Returns a new instance of [PersistConfig] for tests.
-    #[cfg(test)]
```

Comment:
let's put this back and fix whatever WIP thing caused me to do it in the first place :)